A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/SAI/A320_Neo_SAI.js

```diff
@@ -1002,5 +1003,209 @@ class A320_Neo_SAI_PressureIndicator extends HTMLElement {
 }
 customElements.define('a320-neo-sai-pressure-indicator', A320_Neo_SAI_PressureIndicator);
 
+class A320_Neo_SAI_SelfTest extends NavSystemElement {
+    init(root) {
+        this._lastTime = Date.now();
+        this.selfTestElement = this.gps.getChildById("SelfTest");
+        this.complete = false;
```

Comment:
where is this getting set to true?
i see this.complete = true down below but it's in a different class and wouldn't touch this one, right?

Reply:
Just a fragment of code from an earlier version where this flag was set in the extended NavSystemElements class, should have been removed